rasa/shared/core/domain.py

```diff
@@ -521,12 +536,15 @@ def __init__(
         action_names += overridden_form_actions
 
         self.slots = slots
-        self.templates = templates
-        self.action_texts = action_texts or []
         self.session_config = session_config
 
         self._custom_actions = action_names
 
+        self.action_texts = action_texts or []
+        self._user_responses = templates
```

Comment:
Does this mean responses _to_ the user?

Reply:
renamed it. Is it clearer now?

Reply:
The difference is that these responses are defined in the domain by the developer. The bot will send them to the user. The `action_texts` are from stories or rules and will also be sent to the user. However, the developer defines them in an end-to-end fashion within the stories / rules instead of the domain. Does that make sense?